cpp/environment/MatchEnv.cc

```diff
@@ -691,20 +691,23 @@ void MatchEnv::cluster(const freud::locality::NeighborList* env_nlist,
             for (; bond < nlist->getNumBonds() && neighbor_list[2 * bond] == i; ++bond)
             {
                 const size_t j(neighbor_list[2 * bond + 1]);
-                std::pair<rotmat3<float>, BiMap<unsigned int, unsigned int>> mapping
-                    = isSimilar(dj.s[i], dj.s[j], m_threshold_sq, registration);
-                rotmat3<float> rotation = mapping.first;
-                BiMap<unsigned int, unsigned int> vec_map = mapping.second;
-                // if the mapping between the vectors of the environments
-                // is NOT empty, then the environments are similar, so
-                // merge them.
-                if (!vec_map.empty())
+                if (i != j)
```

Comment:
I also commented on this in the other PR: is this exclude_ii intentional?

Reply:
Nope, just forgot to remove.